runtime/compiler/env/j9method.cpp

```diff
@@ -7744,7 +7744,7 @@ TR_J9ByteCodeIlGenerator::runFEMacro(TR::SymbolReference *symRef)
             switch (sourceSig[0])
                {
                case 'L':
-               case '[':
+               case '[': //not sure if for 'Q' should we have something like sourceType = "Qjava/lang/Object"? The same question for line 7758.
```

Comment:
I'm not sure. I would suggest that you add a case for Q, but keep the source type that's produced as "Ljava/lang/Object;" for now, and add another TODO (or maybe TBD).